### DISCUSSION TITLE
Add possibility to filter board by user

### DISCUSSION THREAD

**Discussion by kevin-hoog** [inovex/scrumlr.io#4132] [Proposals]
Would make it easier to present your tickets if you could filter the whole board for a specific user easily.

**Comment by brandstetterm** [comment]
Hey @kevin-hoog, 
Thank you so much for your proposal! I really like your idea. I've written it down and I'll definitely bring it up at our next planning meeting.

**Reply by kevin-hoog** [reply]
Nice! I have actually started coding a proposal for it. My design proposal is to change the participants-list so that clicking on a participant filters the board for that participant. The filter would be local as well so it shouldn't disturb any other user. And the settings is moved to a separate cog-wheel-button. I guess the settings could just be removed since you have access to it through the left menu, and the cogwheel should be changed to a participants icon. Would this work? One possible issue with this design is if the board has a lot of participants, I think there is some mechanic for only showing some participants and not all and then you would not be able to filter on any user.
![Skärmbild 2024-05-20 101748](https://github.com/inovex/scrumlr.io/assets/120627309/6372f5a4-c5af-449a-a351-dbabb9079155)
![Skärmbild 2024-05-20 101808](https://github.com/inovex/scrumlr.io/assets/120627309/8bdb4606-50b1-46c8-b5f3-3f2c204d6ce4)
![Skärmbild 2024-05-20 101825](https://github.com/inovex/scrumlr.io/assets/120627309/50529297-84f6-4b3d-9182-cbd2006c9777)
![Skärmbild 2024-05-20 101837](https://github.com/inovex/scrumlr.io/assets/120627309/0d7d5c23-fb71-4bdb-b4cf-02d1f581d2cc)

**Comment by SelinaBuff** [comment]
Thank you for your idea and input. I find your approach very interesting and would like to create a ticket for our backlog based on this discussion. Our current milestone is to implement our planned Template Page, so we will need to postpone this idea for now.

At the moment, I agree with your concerns regarding Design and the Usability. If we assume there are more than five participants on the board, individual users cannot be easily highlighted and filtered. In the settings dialog, all participants can be seen, but the filter function is hindered because the dialog box visually covers the filtered results. Additionally, I find a settings button next to the users in the top right to be confusing. If I were to recommend a filter icon, it would open a dropdown list of users instead of the settings dialog—similar to GitHub (see screenshot).

<img width="185" alt="Bildschirmfoto 2024-05-23 um 14 36 59" src="https://github.com/inovex/scrumlr.io/assets/116653219/fc38ec25-50dc-4cbd-b08e-01283cb2c339">

**Comment by brandstetterm** [comment]
Here's a proposal from ~ two years ago. I don't know why we didn't follow up on it.

https://github.com/inovex/scrumlr.io/assets/36969812/a5a8c946-9692-476b-9ec9-774f3dbe9ef0

**Reply by Schwehn42** [reply]
yes this is the most intuitive way to do it imo. would only require some refinement for the edge cases (like multiple note authors)
